src/main/kotlin/gg/essential/vigilance/gui/CategoryLabel.kt

```diff
@@ -67,4 +62,11 @@ class CategoryLabel(private val gui: SettingsGui, internal val category: Categor
             setColorAnimation(Animations.OUT_EXP, 0.5f, VigilancePalette.text.toConstraint())
         }
     }
+
+    fun markSelected() {
+        isSelected = true
+        text.animate {
+            setColorAnimation(Animations.OUT_EXP, 0.5f, VigilancePalette.textActive.toConstraint())
+        }
+    }
```

Comment:
For symmetry I think we should also make a `markDeselected` (or have this method accept a `selected: Boolean`) which the current `deselect` calls, and then slap a `@Deprecated` on `select` and `deselect` because we shouldn't be using them any more, they only exist for backwards compatibility, we should always be using `gui.selectCategory` (which will in turn only use the `mark*` method(s)).